plantcv/plantcv/visualize/obj_size_ecdf.py

```diff
@@ -0,0 +1,47 @@
+# Plot Empirical Cumulative Distribution Function for Object Size
+
+import os
+import cv2
+import pandas as pd
+import numpy as np
+from plantcv.plantcv import params, fatal_error
+from plantcv.plantcv._debug import _debug
+from statsmodels.distributions.empirical_distribution import ECDF
+from plotnine import ggplot, aes, geom_point, labels, \
+    scale_color_manual, scale_x_log10
+
+
+def obj_size_ecdf(mask, title=None):
+    """ Plot empirical cumulative distribution for object size based on binary
+    mask
+
+    Inputs:
+    mask  = binary mask
+    title = a custom title for the plot (default=None)
+
+    Returns:
+    fig_ecdf = empirical cumulative distribution function plot
+
+    :param mask: numpy.ndarray
+    :param title: str
+    :return fig_ecdf: plotnine.ggplot.ggplot
+    """
+
+    objects, hierarchy = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_NONE)[-2:]
+    areas = [cv2.contourArea(cnt) for cnt in objects]
+
```

Comment:
```suggestion
# Remove objects with areas < 1px
areas = [i for i in areas if i >= 1.0]

```

If we remove objects with areas < 1 it prevents RuntimeWarnings for log10

Reply:
Good point!